src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorDocumentSynchronizationEndpoint.cs

```diff
@@ -155,7 +163,12 @@ internal SourceText ApplyContentChanges(IEnumerable<TextDocumentContentChangeEve
         {
             foreach (var change in contentChanges)
             {
-                var linePosition = new LinePosition((int)change.Range.Start.Line, (int)change.Range.Start.Character);
+                if (change.Range is null)
+                {
+                    throw new ArgumentNullException(nameof(change.Range), "Range of change should not be null.");
```

Comment:
The change of range falls mainly on the plange.

Reply:
Can't tell if I made a mistake or you just think the phrasing is amusing.

Reply:
Just amused